libs/components/help-inline/src/lib/modules/help-inline/help-inline.component.spec.ts

```diff
@@ -510,28 +506,6 @@ describe('Help inline component', () => {
 
         expect(mockHelpSvc.openHelp).not.toHaveBeenCalled();
       });
-
-      it('should fall back to popover when popoverContent is also specified', async () => {
```

Comment:
This test asserts that both popovers and help keys can be used at the same time, which isn't a pattern we want to support going forward.